std/file.d

```diff
@@ -401,11 +401,18 @@ version (Posix) private void[] readImpl(scope const(char)[] name, scope const(FS
 }
 
 
+// Hack to support unittesting "read" for file handles which do not support GetFileSize:
+version (Windows) private bool readImplTest = false;
+
 version (Windows) private void[] readImpl(scope const(char)[] name, scope const(FSChar)* namez,
-                                          size_t upTo = size_t.max) @safe
+                                          size_t upTo = size_t.max) @trusted
 {
     import core.memory : GC;
     import std.algorithm.comparison : min;
+    import std.conv : to;
+    import std.experimental.checkedint : checked;
+    //import std.stdio;
```

Comment:
Be sure to remove temporary debug code before submitting.

Reply:
The writef's ! Okay, I am going to remove them !

Regarding the highlighted lines in your comment, does "temporary debug code" [also] refer to checkedint, please ?

Reply:
No, though I understand that's a candidate to be refactored away into a common read loop function.